ansible-core/packaging/debian/control

```diff
@@ -31,10 +31,10 @@ Depends:
  ${misc:Depends},
  ${python3:Depends},
  python3-cryptography,
- python3-jinja2 (>= 3.0.0),
+ python3-jinja2 (>= 3.1.0),
  python3-packaging,
- python3-resolvelib (<< 1.1.0),
  python3-resolvelib (>= 0.5.3),
+ python3-resolvelib (<< 2.0.0),
  python3-yaml (>= 5.1)
```

Comment:
Is this file just used for ansible-core 2.19, or also for older ansible-core versions? If the latter, than these changes are a bad idea, since ansible-core < 2.19 has different requirements.

Reply:
Good catch! This is just used for ansible 12 (ansible-core 2.19) as it is a PR for the [`ansible-12`](https://github.com/ansible-community/ppa/tree/ansible-12) branch

Reply:
Ah, I missed that! Thanks for clarifying :)